server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosResponseDTO.java

```diff
@@ -0,0 +1,134 @@
+package com.objectcomputing.checkins.services.kudos;
+
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
+import com.objectcomputing.checkins.services.team.Team;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.Nullable;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+@Introspected
+public class KudosResponseDTO {
+
+    @NotNull
+    private UUID id;
+
+    @NotBlank
+    private String message;
+
+    @NotNull
+    private UUID senderId;
+
+    @NotNull
+    private LocalDate dateCreated;
+
+    @Nullable
+    private LocalDate dateApproved;
+
+    @Nullable
+    private Boolean publiclyVisible;
```

Comment:
Why is this `@Nullable` here, and in `Kudos`, but `@NotNull` in `KudosCreateDTO`?

It's also Nullable in the db migration script `server/src/main/resources/db/common/V108__create_kudos_and_kudos_recipient_table.sql`

🤔